recipe/meta.yaml

```
@@ -25,7 +25,7 @@ requirements:
     - pytorch >=1.4.0
```

The current package uses the `requirements.txt` file to pull in `setup.py:install_requires`. 

```sh
numpy
Pillow
torch>=1.7.1
torchvision>=0.8.2
ttach
tqdm
opencv-python
```

```suggestion
    - pytorch >=1.7.1
```